tests/e2e/configurer/config/constants.go

```diff
@@ -35,4 +35,7 @@ var (
 	// creation in case more pools are added to genesis
 	// in the future
 	PreUpgradePoolId uint64 = 2
+	// Stableswap pool is created after balance pool,
+	// so the pool id should be 3.
+	PreUpgradeStableSwapPoolId uint64 = 3
```

Comment:
This has changed since #4384 

Let's make sure that we are not hard-coding this value. Instead, let's get the resulting pool id from `CreateStableSwapPool` and persist it until we reach `TestStableSwapPostUpgrade`

Can you please make a similar update for `PreUpgradePoolId`?

Reply:
We do something similar with wallet creation where we set it pre-upgrade here:
https://github.com/osmosis-labs/osmosis/blob/7158199261ef7f5be709cd763377e4ecc2effa4d/tests/e2e/configurer/upgrade.go#L159-L163

and then use here in tests:
https://github.com/osmosis-labs/osmosis/blob/7158199261ef7f5be709cd763377e4ecc2effa4d/tests/e2e/e2e_test.go#L1116

We should have similar logic for pool ids because now we create pools with ids 1 to 833 in genesis. So your stabelswap pool number is likely to be 835 or so